tests/data/comments7.py

```diff
@@ -93,9 +93,7 @@ def func():
 
 
 def func():
-    c = call(
-        0.0123, 0.0456, 0.0789, 0.0123, 0.0789, a[-1],  # type: ignore
-    )
+    c = call(0.0123, 0.0456, 0.0789, 0.0123, 0.0789, a[-1],)  # type: ignore
```

Comment:
As an aside, I really don't like the side-effect of #826 that causes this comma to be preserved. It seems like it won't cause much trouble for an already-formatted codebase but I would find it extremely frustrating to clean up in a project that I'm newly formatting.

Reply:
@ambv has some exciting ideas about invisible commas that could fix this.

Reply:
Yeah, we'll fix it in the next beta.

Reply:
@msullivan, while I have you here, if you redo your mypycization of Black for the current state of things I promise to land it very quickly. Sorry the other one went stale.